tinylog-impl/src/main/java/org/tinylog/core/LogEntryValue.java

```diff
@@ -25,6 +25,13 @@
 	 */
 	DATE,
 
+	/**
+	 * Timestamp of issue.
+	 *
+	 * @see LogEntry#getTimestamp()
+	 */
+	TIMESTAMP,
+
```

Comment:
You don't need to add an additional enum value to `LogEntryValue`. The existing enum value ensures already that a log entry will always have a timestamp.